koku/masu/database/trino_sql/reporting_ocpawscostlineitem_daily_summary.sql

```diff
@@ -383,6 +409,8 @@ SELECT aws.uuid as aws_uuid,
         max(aws.blended_cost) * cast({{markup}} as decimal(33,15)) as markup_cost_blended,
         max(aws.savingsplan_effective_cost) as savingsplan_effective_cost,
         max(aws.savingsplan_effective_cost) * cast({{markup}} as decimal(33,15)) as markup_cost_savingsplan,
+        max(aws.calculated_amortized_cost) as calculated_amortized_cost,
+        max(aws.calculated_amortized_cost) * cast({{markup}} as decimal(33,9)) as calculated_amortized_cost,
```

Comment:
This looks wrong, I dont think we should be returning calculated_amortized_cost for both here this one should probably be markup_cost_amortized I'm guessing
```suggestion
        max(aws.calculated_amortized_cost) * cast({{markup}} as decimal(33,9)) as markup_cost_amortized,
```